week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,69 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+  console.log("The ball has shaken!");
+  const randomNumber= Math.random()*20;
+  const randomNumberFloor= Math.floor(randomNumber);
+  const options= ["It is certain",
+    "It is decidedly so",
+    "Without a doubt",
+    "Yes - definitely",
+    "You may rely on it",
+    "As I see it, yes",
+    "Most likely",
+    "Outlook good",
+    "Yes",
+    "Signs point to yes",
+    "Reply hazy, try again",
+    "Ask again later",
+    "Better not tell you now",
+    "Cannot predict now",
+    "Concentrate and ask again",
+    "Don't count on it",
+    "My reply is no",
+    "My sources say no",
+    "Outlook not so good",
+    "Very doubtful"];
+  return `${options[randomNumberFloor]}`;
+}
 
 // The answer should come from shaking the ball
-let answer;
+let answer=shakeBall();
 
 // When checking the answer, we should tell someone if the answer is
 // - very positive
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
+function checkAnswer(answer) {
+  let levelOfPositivity;
+
+  if(answer=="It is certain"|| answer=="It is decidedly so" || answer=="Without a doubt"||answer=="Yes - definitely"||answer=="You may rely on it")
+  {
+    levelOfPositivity="very positive";
```

Comment:
Instead of re-assiging to `"very positive"` how else could you make the function output the level of positivity ?